packages/number-line/configure/src/domain.jsx

```diff
@@ -23,7 +23,13 @@ export class Domain extends React.Component {
 
   change(key, event, value) {
     const { onChange } = this.props;
-    const update = { ...this.props.domain, [key]: value };
+    let update;
+    //Added condition when min and max is same, then it should not update the value
```

Comment:
in this case isn't it better to just return and not call onChange?

```suggestion

if ((key === 'min' && value === this.props.domain.max) || (key === 'max' && value === this.props.domain.min)) {
return;
}

const update = { ...this.props.domain, [key]: value };
onChange(sort(update));
```

Reply:
If we just return, then the new value in the input field remain as it is. It does not get revert back to old value. Thats why calling onChange().